src/coreclr/tests/src/Loader/binding/tracing/AssemblyToLoad.csproj

```diff
@@ -11,4 +11,7 @@
   <ItemGroup>
     <ProjectReference Include="AssemblyToLoadDependency.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Resource.fr-FR.resx" />
+  </ItemGroup>
```

Comment:
Is this a related change?

Reply:
It gives this assembly a satellite assembly (fr-FR culture) that the tests load.